tools/clean_svg.py

```diff
@@ -84,14 +84,21 @@ def round_if_number(value):
 def remove_non_svg_nodes_and_strip_namespace(root):
     if root.tag.startswith(BRACKETS_NAMESPACE):
         root.tag = root.tag[len(BRACKETS_NAMESPACE) :]
+
+    # we cannot remove while iterating on the same iterable,
+    # so we first get the list of nodes to remove, and then
+    # remove them in a second pass
+    cleanup_elems = [
+        elem
+        for elem in root
+        if not elem.tag.startswith(BRACKETS_NAMESPACE)
+        or elem.tag == BRACKETS_NAMESPACE + "metadata"
+    ]
+    for elem in cleanup_elems:
+        root.remove(elem)
+
     for elem in root:
```

Comment:
heh, I think you could've gotten the same effect with:
```
for elem in list(root):
```
which *should* create a list first and then operate on the list instead.

Reply:
indeed, it works